readme.md

```diff
@@ -1,6 +1,6 @@
 # Stopwatch task
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_stop-watch/)
+- [DEMO LINK](https://ColumbusDeveloper.github.io/layout_calendar/)
```

Comment:
please add DEMO link to PR description and here should be a link to [layout_stop-watch]